src/myapps/organising-download-dir.py

```diff
@@ -0,0 +1,119 @@
+# automation.py
+import os
+from os.path import isfile, join
+from pathlib import Path
+from shutil import move
+from typing import List
+import logging
+
+# directory paths
+
+BASE_PATH = Path(os.path.abspath(__file__)).parent.parent.parent.parent.parent.parent
+TARGET_PARENT = "Downloads"  # change target location here; e.g. /home/username/{foobar}
+TARGET_PARENT_PATH = f"{BASE_PATH}/{TARGET_PARENT}"
+
+parent_images_path = f"{TARGET_PARENT_PATH}/images"
+parent_documents_path = f"{TARGET_PARENT_PATH}/documents"
+parent_others_path = f"{TARGET_PARENT_PATH}/others"
+parent_softwares_path = f"{TARGET_PARENT_PATH}/softwares"
+
+# category wise file types
+
+doc_types = (".doc", ".docx", ".txt", ".pdf", ".xls", ".ppt", ".xlsx", ".pptx")
+img_types = (".jpg", ".jpeg", ".png", ".svg", ".gif", ".tif", ".tiff")
+software_types = (".exe", ".pkg", ".dmg", "deb")
+
+
+def get_files(parent_dir) -> List:
+    """Returns a list containing every filenames in the
+    `parent_dir`. Hidden or dot files are ignored
+
+    Arguments:
+        parent_dir {string} -- base path
+
+    Returns:
+        List -- Every filenames found in `parent_dir` (e.g. 'foobar.txt')
+    """
+    return [
+        f
+        for f in os.listdir(parent_dir)
+        if isfile(join(parent_dir, f))
+        and not f.startswith(".")
+        and not f.__eq__(__file__)
+    ]
+
+
+def create_directories(image_path, documents_path, software_path, others_path):
+    """Creates necessary target directories for moving files to
+
+    Arguments:
+        image_path {string} -- the images directory.
+        documents_path {string} -- the documents directory.
+        software_path {string} -- the software/applications directory.
+        others_path {string} -- the directory for files that do not belong to the above three.
+    """
+    dir_dict = {
+        "image_dir": image_path,
+        "docs_dir": documents_path,
+        "software_dir": software_path,
+        "others": others_path,
+    }
+    # print(dir_dict)
+    for k, v in dir_dict.items():
+        if os.path.isdir(v) == True:
+            continue
+        else:
```

Comment:
And as you have the `continue` keyword in the if block, you could (if you wanted to and if you think it's easier to read) remove the else keyword and the code will still execute these next 2 lines.